lib/datadog/tracing/contrib/graphql/configuration/error_extension_env_parser.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Datadog
+  module Tracing
+    module Contrib
+      module GraphQL
+        module Configuration
+          # Parses the environment variable `DD_TRACE_GRAPHQL_ERROR_EXTENSIONS` for error extension names declaration.
+          class ErrorExtensionEnvParser
+            # Parses the environment variable `DD_TRACE_GRAPHQL_ERROR_EXTENSIONS` into an array of error extension names.
+            def self.call(values)
+              return unless values
```

Comment:
Should it be an empty array here?

```suggestion
              return [] unless values
```

P.S does it mean that this branch is not covered in tests?

Reply:
Good call, this is actually unreachable because of this guard: https://github.com/DataDog/dd-trace-rb/blob/aa7e0da03fb5a49d6e2ea56fa48200c0489e8405/lib/datadog/core/configuration/option.rb#L293

I'll remove this line.

Reply:
I added docs explaining this to, for other users of `env_parser`.